ede-php-autoload/class-loader/psr0.el

```diff
@@ -125,17 +124,26 @@ PREFIX is the beginning of the type to complete."
                             namespace
                             "")))
       (setq suggestions (delete-dups
-                         (delete nil
-                                 (mapcar
-                                  #'(lambda (file-name)
-                                      (ede-php-autoload--create-psr-0-suggestions
-                                       file-name
-                                       prefix))
-                                  (ede-php-autoload--gather-relative-subfiles
-                                   directories
-                                   project-root
-                                   relative-path
-                                   (car (last split-prefix)))))))))
+                         (mapcar
+                          #'(lambda (file-name)
+                              (let ((suggestion (ede-php-autoload--create-psr-0-suggestions
+                                                 file-name
+                                                 prefix)))
+                                (ede-php-autoload--make-completion
+                                 suggestion
+                                 ;; A bit of a cop out, but right now it's the best option.
+                                 (if (string-match-p "\\.php$" file-name) :file :directory)
```

Comment:
What about something like `(file-regular-p file-name)`? That's my opinion but the more I avoid tests with regular expression the better I feel.